recipes/DataProfiler/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set name = "dataprofiler" %}
+{% set version = "0.8.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/DataProfiler-{{ version }}.tar.gz
+  sha256: ebd710d9bdd61cd2a66a5719ed3cc6a0442676191de299cdc978a5ec54ada813
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  skip: true  # [py>=38 or py2k or py<37]
```

Comment:
You're skipping everything here which is why this is passing. This should probably be changed back to noarch and the issue resolved.